utilities/translation/tfcomp.js

```diff
@@ -31,6 +31,8 @@ let frFileMissList = [];
 
 const jsonFiles = buildJsonFileArray();
 
+const notSwapFile = (fname) => !fname.includes('.swp');
```

Comment:
I don't actually think this fix works, actually.  You can test it out.  I had limited success.